From a8161b640321ce4159864390b114c2b1362a3990 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 29 Dec 2022 14:21:25 -0800
Subject: [PATCH] Add installation patch

---
 Makefile | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/Makefile b/Makefile
index af9701c..c7dd213 100644
--- a/Makefile
+++ b/Makefile
@@ -141,6 +141,7 @@ MBR_LIB_OBJS=$(MBR_LIBS:=.o)
 LIB_HEADERS=$(LIB_NAMES:=.h)
 DEPEND= makedepend $(CXXFLAGS)
 ALL_EXE=$(ALL:=$(FN_EXTENSION))
+DESTDIR=/
 
 all:	$(ALL)
 
@@ -177,3 +178,14 @@ $(OBJS):
 
 # makedepend dependencies below -- type "makedepend *.cc" to regenerate....
 # DO NOT DELETE
+install:
+	mkdir -p $(DESTDIR)/usr/sbin
+	install -m 0755 gdisk $(DESTDIR)/usr/sbin/gdisk
+	install -m 0755 sgdisk $(DESTDIR)/usr/sbin/sgdisk
+	install -m 0755 cgdisk $(DESTDIR)/usr/sbin/cgdisk
+	install -m 0755 fixparts $(DESTDIR)/usr/sbin/fixparts
+	mkdir -p $(DESTDIR)/usr/share/man/man8
+	install -m 0644 gdisk.8 $(DESTDIR)/usr/share/man/man8/gdisk.8
+	install -m 0644 sgdisk.8 $(DESTDIR)/usr/share/man/man8/sgdisk.8
+	install -m 0644 cgdisk.8 $(DESTDIR)/usr/share/man/man8/cgdisk.8
+	install -m 0644 fixparts.8 $(DESTDIR)/usr/share/man/man8/fixparts.8
-- 
2.38.1

